WalletWasabi.Gui/Controls/WalletExplorer/TransactionViewerViewModel.cs

```diff
@@ -64,12 +67,14 @@ public byte[] PsbtBytes
 		{
 			get => _psbtBytes;
 			set => this.RaiseAndSetIfChanged(ref _psbtBytes, value);
-		}
+		}		
 
-		public TransactionViewerViewModel(WalletViewModel walletViewModel) : base("Transaction", walletViewModel)
+		public TransactionViewerViewModel(Guid walletId) : base("Transaction")
 		{
 			Global = Locator.Current.GetService<Global>();
 
+			Id = walletId;
```

Comment:
This `Id` property is not used anywhere.

Reply:
Used here: https://github.com/zkSNACKs/WalletWasabi/pull/3116/files/a077af522bcdbccf28abb9cd5f74a68a19d7ac03#diff-36af77964c017c1b6ffcbf89774d0aacR30

Reply:
That is where `_walletId` is passed but I mean the property `public Guid Id { get; }` in the `TransactionViewerViewModel` class is not used.